client/src/commands/run.ts

```diff
@@ -128,9 +128,14 @@ async function runCode(selected?: boolean) {
     {
       location: ProgressLocation.Notification,
       title: "SAS code running...",
+      cancellable: typeof session.cancel === "function",
     },
-    () =>
-      session
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Do we _need_ this line? I removed it from your commit and things seem to build without complaint

Reply:
Good catch. Thank you!